src/pages/Login/index.tsx

```diff
@@ -1,75 +1,111 @@
-import React, { useState } from 'react'
-
+import React, { useState, useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { useHistory } from 'react-router-dom'
 import { Link } from 'react-router-dom'
-import { Container, Form, Row, Col } from 'react-bootstrap'
+import { Form, Row, Col } from 'react-bootstrap'
 
 import CustomSvgIcon from '../../components/CustomSvgIcon'
 import starsLady from '../../media/standing-lady.svg'
-import NavBar from '../../components/NavBar'
 import Loader from '../../components/Loader'
+import Message from '../../components/Message'
+import FormContainer from '../../components/FormContainer'
 
 import CustomButton from '../../components/CustomButton'
+import { loginEmployerRequest } from '../../redux/actions/employer'
+import { loginJobseekerRequest } from '../../redux/actions/jobseeker'
+
+import { AppState } from '../../redux/types'
+import { employerInfoFromStorage } from '../../redux/saga/employer'
+import { jobseekerInfoFromStorage } from '../../redux/saga/jobseeker'
 
 const Login = () => {
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
+  const [isLoggedIn, setLoggedIn] = useState(false)
+
+  const dispatch = useDispatch()
+  const history = useHistory()
 
-  // const userInfo = { email: 'duy@abc.com', password: '12345' } // comment out due to husky verification
-  // const error = 'no no no'
-  const loading = false
-  // use useEffect here
-  // if (userInfo) {
-  // 	history.push(redirect)
-  // }
+  const employer = useSelector((state: AppState) => state.employer)
+  const jobseeker = useSelector((state: AppState) => state.jobseeker)
```

Comment:
Can we keep these as local state in this component, cause they're only being used here ?

Reply:
How to differentiate if it's jobseeker or employer logging in, then? The loading and error message come from redux too. I also use those in register page.

Reply:
I meant to use it as local state with `useState` hook instead. But I see you're using it in register also, so it's fine 👍

Reply:
Ok. Maybe I can clean it up though.